src/BookNavigator/book-navigator.js

```diff
@@ -22,63 +19,66 @@ const events = {
 export class BookNavigator extends LitElement {
   static get properties() {
     return {
-      book: { type: Object },
-      pageContainerSelector: { type: String },
-      brWidth: { type: Number },
+      itemMD: { type: Object },
       bookReaderLoaded: { type: Boolean },
       bookreader: { type: Object },
+      bookIsRestricted: { type: Boolean },
       downloadableTypes: { type: Array },
       isAdmin: { type: Boolean },
       lendingInitialized: { type: Boolean },
       lendingStatus: { type: Object },
       menuProviders: { type: Object },
       menuShortcuts: { type: Array },
-      sideMenuOpen: { type: Boolean },
       signedIn: { type: Boolean },
+      loaded: { type: Boolean },
+      /** @type {SharedResizeObserver} */
+      sharedObserver: { type: Object, attribute: false },
+      fullscreenBranding: { type: Object },
```

Comment:
blocker: embedded bookreaders have a title bar -> make sure that we can toggle off the view

(see (possibly deprecating) option `option.ui = embed`)

Reply:
blocker: add `modal` as property